### ISSUE TITLE
"-H" option of test/lib/nvme/overhead throws segmentation fault

### ISSUE THREAD

**Issue by wm8120** [spdk/spdk#236]
Command:
```
sudo ./overhead -s 4096 -t 30 /dev/nvme0n1 -H
```
Output:
```
Starting DPDK 17.11.0 initialization...
[ DPDK EAL parameters: overhead -c 0x1 --file-prefix=spdk0 --base-virtaddr=0x1000000000 --proc-type=auto ]
EAL: Detected 40 lcore(s)
EAL: Auto-detected process type: PRIMARY
EAL: No free hugepages reported in hugepages-1048576kB
EAL: Probing VFIO support...
Measuring overhead for AIO device /dev/nvme0n1.
Initialization complete. Launching workers.
Segmentation fault (core dumped)
```

Basically, this bug can be fixed by adding following two lines in `test/lib/nvme/overhead.c`
```
static int 
register_aio_file(const char *path)
{
...
entry->submit_histogram = spdk_histogram_data_alloc();
entry->complete_histogram = spdk_histogram_data_alloc();
...
}
```

**Comment by jimharris** [comment]
Hi Meng,

Would you like to submit a patch on Gerrithub with this fix?

-Jim

**Comment by wm8120** [comment]
Okay.

**Comment by wanqunintel** [comment]
https://review.gerrithub.io/#/c/403123/

**Comment by dverkamp-intel** [comment]
Fix merged as commit fe6b3de1264616df789aa923eb8e7f88c8aba991, thanks!
